selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/ConnectionsManagement.java

```diff
@@ -97,8 +103,8 @@ public void cancelConnection(String username) {
     info("--Cancel a connection of a user--");
     info("Click on Cancel button");
     searchPeople(username, null, null, null);
-    evt.clickByJavascript(ELEMENT_CONNECTION_CANCEL_BTN.replace("${user}", username));
-    evt.waitForElementNotPresent(ELEMENT_CONNECTION_CANCEL_BTN.replace("${user}", username));
+    $(byText("Cancel Request")).click();
+    $(byText("Cancel Request")).waitUntil(Condition.disappears,Configuration.timeout);
```

Comment:
Please use a variable in SocialLocator instead of using string, to be compliant with other tests.